base/setup/usetup/lang/en-US.h

```diff
@@ -2171,6 +2195,14 @@ MUI_PAGE enUSPages[] =
         SUCCESS_PAGE,
         enUSSuccessPageEntries
     },
+    {
+        BOOT_LOADER_HARDDISK_MBR_PAGE,
+        enUSBootLoaderInstallPageEntries
+    },
+    {
+        BOOT_LOADER_HARDDISK_VBR_PAGE,
+        enUSBootLoaderInstallPageEntries
+    },
```

Comment:
Since it now links to `enUSBootLoaderInstallPageEntries` in both cases, you can add just a single `BOOT_LOADER_INSTALLATION_PAGE`.